etl/steps/data/garden/health/2023-04-25/shared.py

```diff
@@ -0,0 +1,412 @@
+"""Utils (mostly mappings).
+
+The mappings are created from the metadata dictionary provided by the source (a tab in the excel sheet)
+"""
+
+
+# Gender ID mappings
+MAPPING_GENDER_VALUES = {
+    1: "male",
+    2: "female",
+}
+
+# Rename fields
+MAPPING_COLUMN_NAMES = {
+    "country": "country",
+    "wgt": "weight_intra_country",
+    "projwt": "weight_inter_country",
+    "year_calendar": "year",
+    "gender": "gender",
+}
+MAPPING_QUESTION_VALUES = {
+    "q1": {
+        "title": "How much do you know about science?",
+        "answers": {
+            "1": "A lot",
+            "2": "Some",
+            "3": "Not much",
+            "4": "Nothing at all",
+            "99": "DK/Refused",
```

Comment:
Maybe it would be better to use "Don't know" instead of "DK" (idem for the 2020 survey). But if it's too much hassle, ignore it.